src/components/flex/Flex.tsx

```diff
@@ -4,10 +4,11 @@ import theme from './theme.css';
 
 interface FlexProps {
   children: ReactNode;
+  direction?: 'row' | 'column';
```

Comment:
Since this is a new component we have the option to use enums here without introducing a breaking change.
Is this perhaps something we can discuss in the ui guild when a type or an enum is preferred? (Or was this already discussed?)

Reply:
I remember while converting some components to TS, we decided to use strings instead of enums. Importing enums into components were not logical at that point and you will still get code completion with strings.
(https://teamleader.slack.com/archives/C2T7PPV9T/p1652185340106959)

Reply:
I think the decision there was because introducing an enum would be a breaking change for the existing components.